js/statements/alter.js

```diff
@@ -45,6 +45,11 @@ function GenerateAlterTable(options = {}) {
 				Sequence([
 					Keyword("ADD"),
 					Optional("COLUMN"),
```

Comment:
@szarnyasg not sure if COLUMN should be optional here - thoughts?

Reply:
It seems to be working fine:
```
⚪◗ create table t(x int);
⚪◗ alter table t add y varchar;
⚪◗ from t;
┌───────┬─────────┐
│   x   │    y    │
│ int32 │ varchar │
├─────────────────┤
│     0 rows      │
└─────────────────┘
```

Postgres also lists `COLUMN` as optional: https://www.postgresql.org/docs/current/sql-altertable.html
```sql
ADD [ COLUMN ] [ IF NOT EXISTS ] column_name data_type [ COLLATE collation ] [ column_constraint [ ... ] ]
DROP [ COLUMN ] [ IF EXISTS ] column_name [ RESTRICT | CASCADE ]
```

And the grammar definition confirms this:
https://github.com/duckdb/duckdb/blob/main/third_party/libpg_query/grammar/statements/alter_table.y#L137